rest_framework-stubs/decorators.pyi

```diff
@@ -9,17 +10,19 @@ from rest_framework.throttling import BaseThrottle
 from rest_framework.views import APIView, AsView  # noqa: F401
 from typing_extensions import Literal
 
+_VIEW = TypeVar("_VIEW", bound=Callable[..., HttpResponseBase])
```

Comment:
Do you prefer `_VIEW` over `_View`?

Reply:
Just out of habit, no strong preference. I see there are already typevars `_Data` and `_Files` so I can change it. to `_View`.